src/components/timeline/RowYAxes.svelte

```diff
@@ -33,9 +34,56 @@
       let marginWidth = 0;
       const axisClass = 'y-axis';
       gSelection.selectAll(`.${axisClass}`).remove();
+      // TODO: Use this flag to only draw the xRange axis once, can be removed when the TODO below is resolved.
+      xRangeAxisDrawn = false;
 
       for (let i = 0; i < yAxes.length; ++i) {
         const axis = yAxes[i];
+        const xRangeLayers = layers.filter(layer => layer.yAxisId === axis.id && layer.chartType === 'x-range');
+
+        if (xRangeLayers.length === 1 && !xRangeAxisDrawn) {
```

Comment:
Any reason not to just draw all xrange axes for all xrange layers that are `showAsLinePlot`?

Reply:
Also can cause an issue if the first xrange is a normal one and the second one is in the line plot mode